libs/designer-ui/src/lib/displayText/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { Link } from '@fluentui/react';
+
+export interface DisplayTextEditorProps {
+  displayTextText: string;
+  relativeLinkText?: string;
+  relativeLink?: string;
+  openRelativeLink?: (relativeLink: string) => void;
+}
+
+export const DisplayTextEditor: React.FC<DisplayTextEditorProps> = ({
+  displayTextText,
+  relativeLinkText,
+  relativeLink,
+  openRelativeLink,
+}): JSX.Element => {
+  return (
+    <div className="msla-displayText-editor-container">
+      {displayTextText ?? ''}
+      <Link className="msla-workflow-parameters-link-text" onClick={() => openRelativeLink?.(relativeLink ?? '')}>
+        {relativeLinkText}
+      </Link>
+    </div>
+  );
+};
```

Comment:
This is quite specific logic on what this editor does, and I'm wondering how extensible it is. Namely, it will only has the option to show some text and follow it with a link. Although I wouldn't say its blocking for this PR, I think a more relevant editor would to create a markdown editor, where we let this read-only editor be something that the user can customize through the manifest

Reply:
I had to move away from the Markdown Editor because I need to use a specific logic for vscode extension. I found out href does not work in vscode when it's relevant path, so I have to use callback for this link OnClick. I wasn't able to make Markdown to do this, but I'm open to suggestions.

Reply:
Yes, Elaina and I discussed this.  I think overall the idea of having a Markdown component that can be render based on the manifest is quite good. However, in this specific scenario in which we want to render links is different.
For web browsers and web pages in general it works fine. The browser will identify the markdown link and open the page in a new tab.
For vscode extension it will do the same thing, open a tab in the web browser, however, that's not the desired behavior. What is needed in vscode is to open a new vscode tab and that requires a callback function. Also, despite that we are using webviews that doesn't mean that it's an actual web browser being rendered inside vscode. 
I can explain this further if anyone else is interested

Reply:
yeah Im thinking for this here we can have some placeholder text here to encorporate the action needed in the vscode command

Reply:
that being said, I'm starting to like more and more the idea of using a custom editor for this case and moving it outside of the designer?